app/assets/stylesheets/tables-report.css.scss

```diff
@@ -3,6 +3,16 @@
 @forward 'usa-table';
 @forward 'usa-alert';
 
+$secondary-color: #112e51;
+$light-gray: #f3f3f3;
+$white: #fff;
+
+h1,
+h2,
+h3 {
```

Comment:
Should we be overriding all headers like this or creating a new class to use in places we want to use a different color?

Reply:
This stylesheet is only pulled into report emails. I'm open to changing this, but for instance, it doesn't affect the `h2` on http://localhost:3000/rails/mailers/user_mailer/reset_password_instructions_with_pending_gpo_letter